src/constants.js

```diff
@@ -131,11 +131,6 @@ if (window.require) {
  */
 export const IPC_RENDERER = ipcRenderer;
 
-/**
- * Flag to hide/show elements after ledger integration is done
- */
-export const LEDGER_ENABLED = true;
-
```

Comment:
This feature flag was from before ledger was enabled, just removed it since it does not seem useful anymore.